types/govuk-frontend/dist/govuk/common/normalise-string.d.ts

```diff
@@ -0,0 +1,22 @@
+import { type SchemaProperty } from "./index.js";
+
+/**
+ * Normalise string
+ *
+ * 'If it looks like a duck, and it quacks like a duck…' 🦆
+ *
+ * If the passed value looks like a boolean or a number, convert it to a boolean
+ * or number.
+ *
+ * Designed to be used to convert config passed via data attributes (which are
+ * always strings) into something sensible.
+ *
+ * @internal
```

Comment:
this isn't enforced by the compiler, but ... are you intending to include so many functions marked internal?

Reply:
Yeah, rather than cause issues for internal GOV.UK teams I've removed `@private` but kept `@internal`

It's a balance between the [public docs](https://frontend.design-system.service.gov.uk) versus JSDoc generated [internal docs](https://govuk-frontend-review.herokuapp.com/docs/javascript/)

Thanks for looking closely though, appreciate it